vms/platformvm/network/gossip_test.go

```diff
@@ -7,69 +7,29 @@ import (
 	"errors"
 	"testing"
 
-	"github.com/prometheus/client_golang/prometheus"
-	"github.com/stretchr/testify/require"
-	"go.uber.org/mock/gomock"
-
 	"github.com/ava-labs/avalanchego/ids"
 	"github.com/ava-labs/avalanchego/utils/logging"
 	"github.com/ava-labs/avalanchego/vms/platformvm/txs"
-	"github.com/ava-labs/avalanchego/vms/platformvm/txs/mempool/mempoolmock"
+	pmempool "github.com/ava-labs/avalanchego/vms/platformvm/txs/mempool"
 	"github.com/ava-labs/avalanchego/vms/txs/mempool"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/stretchr/testify/require"
 )
 
 var errFoo = errors.New("foo")
 
 // Add should error if verification errors
 func TestGossipMempoolAddVerificationError(t *testing.T) {
 	require := require.New(t)
-	ctrl := gomock.NewController(t)
 
 	txID := ids.GenerateTestID()
 	tx := &txs.Tx{
 		TxID: txID,
 	}
 
-	mempool := mempoolmock.NewMempool(ctrl)
-	txVerifier := testTxVerifier{err: errFoo}
-
-	mempool.EXPECT().Get(txID).Return(nil, false)
-	mempool.EXPECT().GetDropReason(txID).Return(nil)
-	mempool.EXPECT().MarkDropped(txID, errFoo)
-
-	gossipMempool, err := newGossipMempool(
-		mempool,
-		prometheus.NewRegistry(),
-		logging.NoLog{},
-		txVerifier,
-		testConfig.ExpectedBloomFilterElements,
-		testConfig.ExpectedBloomFilterFalsePositiveProbability,
-		testConfig.MaxBloomFilterFalsePositiveProbability,
-	)
+	mempool, err := pmempool.New("", prometheus.NewRegistry(), nil)
 	require.NoError(err)
-
-	err = gossipMempool.Add(tx)
-	require.ErrorIs(err, errFoo)
-	require.False(gossipMempool.bloom.Has(tx))
-}
-
-// Add should error if adding to the mempool errors
-func TestGossipMempoolAddError(t *testing.T) {
```

Comment:
This test seems to just test the invariants of the mempool (too big/max size/etc) so I removed it since similar coverage exists elsewhere in this package